sdk/keyvault/keyvault-certificates/tests/utils/recorder.ts

```diff
@@ -0,0 +1,434 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import fs from "fs-extra";
+import nise from "nise";
+import { retry as realRetry } from "./retry";
+import { isNode as coreIsNode, delay as coreDelay } from "@azure/core-http";
+import queryString from "query-string";
+import * as dotenv from "dotenv";
+dotenv.config({ path: "../.env" });
+
+export function isBrowser(): boolean {
+  return typeof window !== "undefined";
```

Comment:
Is it better to do this or just say !isNode()?

Reply:
Yes! I agree. However: This recorder will be removed soon. This feedback is probably relevant in the new core recorder too. Should I still do the change!